src/main/java/com/mindee/ParseParameter.java

```diff
@@ -9,12 +9,12 @@
  */
 @Value
 @Builder
+@Deprecated
```

Comment:
@vivekpaddy As we'll be releasing this in a major version, better to simply remove any deprecated classes and methods.